docs/user-guide/sdks-using.md

```diff
@@ -40,32 +42,71 @@ If you download the Python SDK packages from Zowe.org, the downloaded folder con
 pip install <fileName>.whl
 ```
 
+### Java
+
+Requires Java runtime version 11 and above.  
+
 ## Getting started
 
 To get started, import the SDK packages to your project. You can pull the packages from an online registry, or download the packages from Zowe.org to install locally.
 
-### Install SDK from online registry
+### Install Node.js from online registry
 
-Pull the packages from an online registry such as npm or PyPi.
+Pull the packages from an online registry such as npm.
 
 **Follow these steps:**
 
 1. In command-line window, navigate to your project directory. Issue the following command to install a package from the registry:
 
    - To import a Node.js package: `npm install <PackageName>`
-   - To import a Python package: `pip install <PackageName>`
 
-   *where* `<packageName>` is the name of the SDK package that you want to install, such as `zos-files-for-zowe-sdk`.
+     *where* `<packageName>` is the name of the SDK package that you want to install, such as `zos-files-for-zowe-sdk`.
 
-    The packages are installed. Node packages are defined in `package.json` in your project. Python packages are installed by default to `$PYTHONPATH/Lib/site-packages` (Linux) or to the Python folder in your local `/AppData` folder (Windows).
+     The packages are installed. Node packages are defined in `package.json` in your project. 
 
 2. **(Optional)** You might want to automatically update the SDK version when updates become available, or you might want to prevent automatic updates.
 
     - To define the versioning scheme for Node packages, use [semantic versioning](https://docs.npmjs.com/about-semantic-versioning).
 
+### Install Java SDK from online registry
```

Comment:
please updated the section header to: Install the Java SDK from an online registry

Reply:
done